openssl/src/ssl/mod.rs

```diff
@@ -774,12 +774,12 @@ impl SslContextBuilder {
     /// curve.
     ///
     /// Requires the `v102` feature and OpenSSL 1.0.2.
-    #[cfg(all(feature = "v102", ossl102))]
+    #[cfg(all(feature = "v102", any(ossl102, libressl)))]
```

Comment:
set_ecdh_auto has been there since at least libressl 2.1.2, for what it's worth.

Having features specific to libressl, wouldn't that make things harder for downstreams to be compatible with both libressl and openssl?